akka-actor/src/main/scala/akka/event/LoggerMailbox.scala

```diff
@@ -43,6 +43,5 @@ private[akka] class LoggerMailbox(owner: ActorRef, system: ActorSystem)
         envelope = dequeue
       }
     }
-    super.cleanUp(owner, deadLetters)
```

Comment:
@ktoso is common sense, testing is a good practice and everything should be tested but there are cases where mathematical logic should prevail.

This case I think is one of that, when you wrote the original code and having all the data of when a code is reproduced should you still have a robot tell you a human what your code is supposed to do?

The call, without any doubt according to @patriknw is harmless and also according to the use case ... useless, shouldn't be there.